### PR TITLE
Update settings.dist.py to force_auth for SAML2

### PR DESCRIPTION
**Pull request by italvi**
Make SSO over SAML2 possible (#6092)

### CODE REVIEW

File: dojo/settings/settings.dist.py [hunk 1]
```diff
@@ -848,7 +850,7 @@ def saml2_attrib_map_format(dict):
                 'name_id_format': saml2.saml.NAMEID_FORMAT_TRANSIENT,
                 'want_response_signed': False,
                 'want_assertions_signed': True,
-                'force_authn': True,
+                'force_authn': env('DD_SAML2_FORCE_AUTH'),
```
Comment (dsever) [inline dojo/settings/settings.dist.py:853]: Do not use env variables directly, config should support config py files as well 

Please take a look:

https://github.com/italvi/django-DefectDojo/blob/patch-1/dojo/settings/settings.dist.py#L826

**Review by dsever — CHANGES_REQUESTED** [review]
Please take a look at my comment

File: dojo/settings/settings.dist.py (hunk 1 above)
Comment (dsever) [reply dojo/settings/settings.dist.py]: Actually this is my mistake :/ can you fix this as well?

**Review by StefanFl — APPROVED** [review]
Hi @italvi, thank you for your contribution, lgtm

**Review by dsever — APPROVED** [review]
Looks ok

**Comment by nobletrout** [comment]
Is this a breaking change for existing SAML logins?

**Comment by dsever** [comment]
@nobletrout  do you have some issues after the upgrade?

**Comment by nobletrout** [comment]
No I haven't upgraded yet was reading release notes and raising a concern

**Comment by italvi** [comment]
The default value is still True, therefore it should not have any impact on existing SAML Logins.

**Pull request merged**